[2022-08-29 15:38:28,595] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: h1b_lca_data_v3.entry_task scheduled__2021-01-10T06:00:00+00:00 [queued]>
[2022-08-29 15:38:28,617] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: h1b_lca_data_v3.entry_task scheduled__2021-01-10T06:00:00+00:00 [queued]>
[2022-08-29 15:38:28,618] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-08-29 15:38:28,619] {taskinstance.py:1239} INFO - Starting attempt 19 of 19
[2022-08-29 15:38:28,620] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-08-29 15:38:28,636] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): entry_task> on 2021-01-10 06:00:00+00:00
[2022-08-29 15:38:28,647] {standard_task_runner.py:52} INFO - Started process 11618 to run task
[2022-08-29 15:38:28,653] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'h1b_lca_data_v3', 'entry_task', 'scheduled__2021-01-10T06:00:00+00:00', '--job-id', '1518', '--raw', '--subdir', 'DAGS_FOLDER/upload_h1b_data_to_gcp.py', '--cfg-path', '/tmp/tmpbn2stcqj', '--error-file', '/tmp/tmpp2dnddvq']
[2022-08-29 15:38:28,655] {standard_task_runner.py:77} INFO - Job 1518: Subtask entry_task
[2022-08-29 15:38:28,739] {logging_mixin.py:109} INFO - Running <TaskInstance: h1b_lca_data_v3.entry_task scheduled__2021-01-10T06:00:00+00:00 [running]> on host efb24f1b01c6
[2022-08-29 15:38:28,817] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=h1b_lca_data_v3
AIRFLOW_CTX_TASK_ID=entry_task
AIRFLOW_CTX_EXECUTION_DATE=2021-01-10T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-01-10T06:00:00+00:00
[2022-08-29 15:38:28,820] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-29 15:38:28,821] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo 0']
[2022-08-29 15:38:28,839] {subprocess.py:85} INFO - Output:
[2022-08-29 15:38:28,843] {subprocess.py:89} INFO - 0
[2022-08-29 15:38:28,844] {subprocess.py:93} INFO - Command exited with return code 0
[2022-08-29 15:38:28,875] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=h1b_lca_data_v3, task_id=entry_task, execution_date=20210110T060000, start_date=20220829T153828, end_date=20220829T153828
[2022-08-29 15:38:28,908] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-08-29 15:38:28,950] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
